app/Http/Controllers/ASController.php

```
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+/**
+ * Stub controller for CI - the real implementation is gitignored.
+ */
+class ASController extends Controller
+{
+    public function index($id)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)